code/game/jobs/jobs.dm

```diff
@@ -93,7 +92,6 @@ GLOBAL_LIST_INIT(whitelisted_positions, list(
 	"Blueshield",
 	"Nanotrasen Representative",
 	"Barber",
```

Comment:
```suggestion
	"Barber"
```
I believe we need to drop the comma here yes?

Reply:
Its a styling thing, but DM lists still compile with trailing commas